aio.core/aio/core/functional/process.py

```diff
@@ -0,0 +1,25 @@
+
+from concurrent import futures
+from typing import AsyncIterator, Callable, Iterable
+
+
+async def async_map(
+        fun: Callable,
+        iterable: Iterable,
+        fork: bool = False) -> AsyncIterator:
+    """Asynchronously map sync fun with an iterable, yielding results as
+    available.
+
+    By default a `ThreadPoolExecutor` will be used, setting `fork` to
+    `True` will instead use a `ProcessPoolExecutor` to fork the fun to
```

Comment:
fun

Reply:
i guess im just trying to put some fun into coding - ill remove/expand 8/